src/warc2zim/converter.py

```diff
@@ -64,11 +57,8 @@
 # HTML mime types
 HTML_TYPES = ("text/html", "application/xhtml", "application/xhtml+xml")
 
-# external sw.js filename
-SW_JS = "sw.js"
-
 # head insert template
-HEAD_INSERT_FILE = "sw_check.html"
+HEAD_INSERT_FILE = None
```

Comment:
Any reason to keep all this stuff about head inserts? It looks they are not used anymore, but do you keep them because you will need them at a later stage? If we are mostly sure they won't be needed, I would get rid of them (we can always come back to this commit to re-add them if needed). I generally dislike dead code which often deprecates very fast and gives a false sense of "we just have to reactivate the feature".